go/pkg/fakes/server.go

```diff
@@ -171,7 +171,8 @@ func (e *TestEnv) Set(cmd *command.Command, opt *command.ExecutionOptions, res *
 		}
 	}
 
-	root, inputs, _, err := e.Client.GrpcClient.ComputeMerkleTree(cmd.ExecRoot, cmd.InputSpec, e.Client.FileMetadataCache)
+	execRoot, workingDir, remoteWorkingDir := cmd.ExecRoot, cmd.WorkingDir, cmd.RemoteWorkingDir
```

Comment:
Can you add some verification here to make sure RemoteWorkingDir has the same number of levels as WorkingDir?

Reply:
Added #levels verification to `command.Validate` and triggered command.Validate within `server.TestEnv.Set`